config/locales/security_schemes/en.yml

```diff
@@ -2,3 +2,11 @@ en:
   security_schemes:
     index:
       no_security_scheme: "There are no security schemes, create one !"
+  activerecord:
```

Comment:
As your validator are now unrelated to a specific model, I think you should use this YAML hierarchy:

```
activerecord:
  errors:
    messages:
      value_must_be_valid_json: ...
      value_must_be_hash: ...
```

Source: https://guides.rubyonrails.org/i18n.html#error-message-scopes

Reply:
Hmm… I used what rails gave me. :thinking:

Reply:
Ok, I understood